unit-tests/shared/src/test/scala/javalib/util/DefaultFormatterTest.scala

```diff
@@ -196,11 +196,12 @@ class DefaultFormatterTest {
         classOf[UnsupportedEncodingException],
         new Formatter(notExist, "ISO 1111-1")
       )
-    } finally if (notExist.exists()) {
-      // Fail on RI on Windows, because output stream is created and
-      // not closed when exception thrown
-      assertTrue(notExist.delete())
-    }
+    } finally
```

Comment:
I wish the previous style could stay, it was nice and really readable

Reply:
I think this one is designed that way because it works for braceless syntax. I can try and figure something out but since this is pretty much a one off it may change more with an option.